### PR TITLE
feat: added populate example

### PR DESCRIPTION
**Pull request by uzielflorida**

### CODE REVIEW

File: examples/populate/mintAndEmitLoans.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    abi: erc721AbiWithMint,
+    publicClient,
+    walletClient,
+    address: collection,
+  });
+
+  const tokensToMint = (
+    await Promise.all(
+      new Array(CURRENCIES.length * CONTRACTS.length * NFT_AMOUNT)
+        .fill(0)
```
Comment (mishuagopian) [inline examples/populate/mintAndEmitLoans.ts:69]: You can delete this, right?

File: examples/populate/mintAndEmitLoans.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        .map((_, i) => BigInt(COLLECTION_TOKEN_ID_BASE[collection] + TOKEN_ID_TO_MINT_FROM + i))
```
Comment (mishuagopian) [inline examples/populate/mintAndEmitLoans.ts:70]: I'd remove this map and create the constant in the next one

File: examples/populate/mintAndEmitLoans.ts [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            await erc721.read.ownerOf([tokenIdToMint]);
+          } catch {
+            return [tokenIdToMint, true] as const;
+          }
+          return [tokenIdToMint, false] as const;
+        }),
+    )
+  )
+    .filter(([, isMinted]) => isMinted)
+    .map(([tokenIdToMint]) => tokenIdToMint);
```
Comment (mishuagopian) [inline examples/populate/mintAndEmitLoans.ts:82]: No need to do this

File: examples/populate/mintAndEmitLoans.ts [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+  for (const tokenIdToMint of tokensToMint) {
```
Comment (mishuagopian) [inline examples/populate/mintAndEmitLoans.ts:84]: this would be
```suggestion
  for (const [tokenIdToMint, _] of tokensToMint) {
```

Comment (mishuagopian) [reply examples/populate/mintAndEmitLoans.ts:84]: I'd event remove the filter and just do an if here

File: examples/populate/mintAndEmitLoans.ts (hunk 1 above)
Comment (uzielflorida) [reply examples/populate/mintAndEmitLoans.ts:69]: fill cannot be deleted otherwise array is empty

**Review by mishuagopian — APPROVED** [review]

**Pull request merged**